bin/kubectl-crossplane-krew-wrapper

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+# Wrapper to unify all crossplane-cli plugin commands
+# for use by krew
+
+set -e
+BASEDIR=$(dirname $(realpath "$0"))
+
+PATH="${BASEDIR}:${PATH}" exec $BASEDIR/kubectl-crossplane "$@"
```

Comment:
Hmm I'm not super familiar with the `krew` idioms yet, but it seems like calling this would always result in `Unknown command`. What am I missing?

Reply:
The reasoning behind this is here, https://github.com/crossplaneio/crossplane-cli/issues/32#issue-544715499, based on a conversation with the Krew authors.

Reply:
Updated this based on a review session with @suskin